setup.py

```diff
@@ -5,9 +5,11 @@
     version="0.2.2",
     packages=find_packages(),
     include_package_data=True,
+    python_requires='>=3.8',
     install_requires = [
         "Click",
         "metadatatools @ git+https://git@github.com/IslasGECI/metadata_tools@v0.2.2",
+        "pymc3",
```

Comment:
Será posible quitar `numpy` y `pandas` del Dockerfile y agregarlos aquí:
```suggestion
        "numpy",
        "pandas",
        "pymc3",    
```